Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -1157,13 +1161,7 @@ - (BOOL)isProductionApp {
                              @"most likely a Dev profile.");
   }
 
-#if TARGET_OS_IOS || TARGET_OS_TV
   NSString *apsEnvironment = [plistMap valueForKeyPath:kEntitlementsAPSEnvironmentKey];
-#elif TARGET_OS_OSX
-  NSDictionary *entitlements = [plistMap valueForKey:kEntitlementsKeyForMac];
```

Comment:
Just wanted to confirm that `plistMap` does contain a value for `valueForKeyPath:kEntitlementsAPSEnvironmentKey` for macOS still.

Reply:
Yeah, we redefined this key at line 60 for macOS

#if TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_WATCH
static NSString *const kEntitlementsAPSEnvironmentKey = @"Entitlements.aps-environment";	
#else	
static NSString *const kEntitlementsAPSEnvironmentKey =
    @"Entitlements.com.apple.developer.aps-environment";
#endif